src/app/registration/components/RegistrationSection.tsx

```diff
@@ -0,0 +1,61 @@
+import { IconButton } from '@chakra-ui/button';
+import { Checkbox } from '@chakra-ui/checkbox';
+import { useClipboard } from '@chakra-ui/hooks';
+import { Box, Heading, HStack, Text } from '@chakra-ui/layout';
+import { useToast } from '@chakra-ui/toast';
+import { useCallback, useEffect } from 'react';
+import { IoCopyOutline } from 'react-icons/io5';
+
+import { Seat } from '../../../shared/fetchers';
+import { SeatInfo } from '../../content/components/Seats';
+
+type Props = {
+  section: string;
+  crn: string;
+  seats?: Seat;
+  selected: boolean;
+  handleChange: ({ crn, seats, selected }: { crn: string; seats?: Seat; selected: boolean }) => void;
+};
+
+export function RegistrationSection({ section, crn, seats, selected, handleChange }: Props) {
+  const { hasCopied, onCopy } = useClipboard(crn);
+  const toast = useToast();
+
+  useEffect(() => {
+    hasCopied && toast({ status: 'info', title: `Copied ${crn} to clipboard!` });
+  }, [crn, hasCopied, toast]);
+
+  const onChange = useCallback(() => {
+    handleChange({ crn, seats, selected });
+  }, [crn, handleChange, seats, selected]);
+
+  return (
+    <Box py="2">
+      <HStack justifyContent="space-between">
+        <Heading size="md" as="h3" mt="2">
+          {section}{' '}
+          <Text as="span" fontWeight="normal" ml="2" color="rgb(155, 155, 155)">
+            {crn}
+            <IconButton
+              icon={<IoCopyOutline />}
+              onClick={onCopy}
+              aria-label="copy"
+              colorScheme="white"
+              color="black"
+              _focus={{
+                outline: 'none',
+              }}
```

Comment:
```suggestion
              _focus={{
                outline: 'default',
              }}
```

It is not recommended to remove focus on elements. When a user uses the keyboard to navigate they will be uncertain what is focus if the focus is removed. I think keeping the default focus, or styling the focus is the best option. 

You can read more here: https://webaim.org/techniques/keyboard/

Reply:
thoughts on this solution for a11y? Having the border doesn't look great which is why I removed it so I'm trying some alternatives
<img width="388" alt="Screen Shot 2021-06-06 at 3 53 22 PM" src="https://user-images.githubusercontent.com/53020925/120942671-5c17c380-c6df-11eb-88c5-06a2f7449004.png">

Reply:
I don't think that is enough of a difference for users. The hard thing is is that aesthetic doesn't override accessibility. 

Can you make the icon bigger or more thick when focused? Ideally for convention, developers should leave the outline on focus. Can you try making the outline a little smaller so it doesn't overlap with the CRN?

Reply:
Making it bigger on focus looks nice. I know I'm jumpng hurdles around not having the outline, but it's tough to make it look nice. And yeah, 100% accessibility over aesthetic but if there's a perfect balance between the 2 that's ideal. Of course if there isn't, the outline will be fine.